run.sh

```diff
@@ -29,11 +29,22 @@ if [ -n "$DEPLOY" ]; then
 else
   # its a build!
   export ACTION="build"
-  export ACTION_URL=$WERCKER_BUILD_URL
+  export ACTION_URL=$WERCKER_RUN_URL="https://app.wercker.com/APIPCS/$WERCKER_APPLICATION_NAME/runs/build/$WERCKER_BUILD_ID"
 fi
 
-export MESSAGE="<$ACTION_URL|$ACTION> for $WERCKER_APPLICATION_NAME by $WERCKER_STARTED_BY has $WERCKER_RESULT on branch $WERCKER_GIT_BRANCH"
-export FALLBACK="$ACTION for $WERCKER_APPLICATION_NAME by $WERCKER_STARTED_BY has $WERCKER_RESULT on branch $WERCKER_GIT_BRANCH"
+export WERCKER_SHORT_BUILD_ID=${WERCKER_BUILD_ID:0:5}
+
+if [ "$WERCKER_RESULT" = "failed" ]; then
+  export RESULT="Failed"
+fi
+
+if [ "$WERCKER_RESULT" = "passed" ]; then
+  export RESULT="Sucess"
+fi
+
+export GIT_COMMIT_URL="https://$WERCKER_GIT_DOMAIN/$WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY/commit/$WERCKER_GIT_COMMIT"
+export MESSAGE="$RESULT: #<$ACTION_URL|$WERCKER_SHORT_BUILD_ID> for $WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY by $WERCKER_STARTED_BY on branch <$GIT_COMMIT_URL|$WERCKER_GIT_BRANCH>"
+export FALLBACK="$ACTION for $WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY by $WERCKER_STARTED_BY has $WERCKER_RESULT on branch $WERCKER_GIT_BRANCH"
 export COLOR="good"
 
 if [ "$WERCKER_RESULT" = "failed" ]; then
```

Comment:
we're setting the MESSAGE and FALLBACK here again for the failed builds (it's done for both scenarios a few lines above)